hw/xilinx/ips/hpc/xlnx_axi_clock_converter/config.tcl

```diff
@@ -0,0 +1,11 @@
+# Author: Manuel Maddaluno <manuel.maddaluno@unina.it>
+# Description: AXI Clock Converter - Needed to link the AXI crossbar (in general the entire SoC 250 MHz) to the DDR MIG (300 MHz)
+
+create_ip -name axi_clock_converter -vendor xilinx.com -library ip -version 2.1 -module_name $::env(IP_NAME)
+
+set_property -dict [list \
+    CONFIG.ID_WIDTH {2} \
```

Comment:
for ID_WIDTH and DATA_WIDTH, use config-based envvar, like in https://github.com/MaistoV/UninaSoC/blob/main/hw/xilinx/ips/embedded/xlnx_jtag_axi/config.tcl#L14

Reply:
Agree

Reply:
have you tested this? Vivado was complaining when using envvars in lists TCL. 

I had to work around this, see https://github.com/MaistoV/UninaSoC/blob/main/hw/xilinx/ips/embedded/xlnx_jtag_axi/config.tcl#L14

Reply:
I was able to generate a bitstream with this. But I can change it as you say

Reply:
Maybe is just a issue of (my) version 2022.2, let's have a look during next meeting